### PR TITLE
sts tests code refactor

### PR DESCRIPTION
**Pull request by rakeshgm**
adding just the helper functions for now. 
also removing unused imports 
logs are not needed as there is no change in the scripts. 

Signed-off-by: rakeshgm <rakeshgm@redhat.com>

### CODE REVIEW

**Comment by rakeshgm** [comment]
this is still a work in progress code.

File: rgw/v2/tests/s3_swift/resuables/sts.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+from v2.lib.rgw_config_opts import ConfigOpts
+from v2.lib.s3.auth import Auth
+
+log = logging.getLogger()
+TEST_DATA_PATH = None
+
+
+def add_sts_config_to_ceph_conf(
+    ceph_config_set, rgw_service, sesison_encryption_token="abcdefghijklmnoq"
+):
```
Comment (psathyan) [inline rgw/v2/tests/s3_swift/resuables/sts.py:20]: Can we have docStrings for the new methods?

File: rgw/v2/tests/s3_swift/resuables/sts.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+def create_role(iam_client, policy_document, role_name):
+    log.info("creating role")
+    role_response = iam_client.create_role(
+        AssumeRolePolicyDocument=policy_document,
+        Path="/",
+        RoleName=role_name,
+    )
+    log.info("create_role_response")
+    log.info(role_response)
```
Comment (psathyan) [inline rgw/v2/tests/s3_swift/resuables/sts.py:50]: IMO, we could avoid ln#43, and do it
```
log.info(f"Response of create_role \n{role_response}")
```

File: rgw/v2/tests/s3_swift/resuables/sts.py (hunk 1 above)
Comment (rakeshgm) [reply rgw/v2/tests/s3_swift/resuables/sts.py:20]: @psathyan doc strings added

File: rgw/v2/tests/s3_swift/resuables/sts.py (hunk 2 above)
Comment (rakeshgm) [reply rgw/v2/tests/s3_swift/resuables/sts.py]: done.

**Pull request merged**